components/emoji_picker/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as DtEmojiPicker } from './emoji_picker.vue';
+export {} from './emoji_picker_constants';
```

Comment:
Add the constants that need to be exported, right now, for stories you might not need it, but when we separate the library from storybook, those constants are going to be needed, and maybe someone else needs the constants, so please export them.